CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - `LiveConfig.minTimeshiftBufferDepth` to control the minimum buffer depth of a stream needed to enable time shifting.
 - `Player.buffer` to control buffer preferences and to query the current buffer state
+- `DownloadFinishedEvent` to indicate once a download was finished
```

Comment:
nit: What do you think of rewording this a little bit? It flows better for me this way.

```suggestion
- `DownloadFinishedEvent` to signal when the download of specific content has finished
```